datalad/distributed/create_sibling_ria.py

```diff
@@ -84,6 +84,26 @@ class CreateSiblingRia(Interface):
     The store's base path is expected to not exist, be an empty directory,
     or a valid RIA store.
 
+    RIA URL format
+    ~~~~~~~~~~~~~~
+
+    Interactions with new or existing RIA stores require RIA URLs to identify
+    the store or specific datasets inside of it.
+
+    The general structure of a RIA URL pointing to a store takes the form
+    'ria+[scheme]://<storelocation>' (e.g.,
+    ria+ssh://[user@]hostname:/absolute/path/to/ria-store, or
+    ria+file:///absolute/path/to/ria-store)
+
+    The general structure of a RIA URL pointing to a dataset in a store (for
+    example for cloning) takes a similar form, but appends either the datasets
+    UUID or a ~ symbol followed by the dataset's alias name:
+    'ria+[scheme]://<storelocation>#<dataset-UUID>' or
+    'ria+[scheme]://<storelocation>#~<aliasname>'.
+    In addition, specific version identifiers can be appended to the URL with an
```

Comment:
Citing the original issue:

>  Adding @something to a working URL seems to look for a branch with that name, seems not to work for other tree-ish (commit, tag).

Does it only work for branches? If if only works for branches, should that be made clear?

Reply:
True. It's passed onto the `--branch` option to `git clone`, which considers branches and tags.

Reply:
Would you mind pushing a clarification?